beacon-chain/rpc/proposer_server.go

```diff
@@ -74,6 +74,18 @@ func (ps *ProposerServer) PendingAttestations(ctx context.Context, req *pb.Pendi
 	if err != nil {
 		return nil, fmt.Errorf("could not retrieve pending attestations from operations service: %v", err)
 	}
+	// Remove any attestation from the list if their slot is before the start of
+	// the previous epoch. This should be handled in the operationService cleanup
+	// method, but we should filter here in case it wasn't yet processed.
+	boundary := helpers.StartSlot(helpers.PrevEpoch(beaconState))
+	var attsInBoundary []*pbp2p.Attestation
```

Comment:
by definition boundary is a slot, we want something like `attsSinceLastEpoch` or better : )